### ISSUE TITLE
TimeoutException when installing as a service, to start after installation

### ISSUE THREAD

**Issue by russcam** [elastic/windows-installers#135]
A common installation issue reported by users happens when installing Elasticsearch as a Windows Service, and setting the service to start as part of the installation, but the service is never seen to start. This manifests itself in the install logs similar to

```
Calling custom action Elastic.Installer.Msi!Elastic.Installer.Msi.Elasticsearch.CustomActions.Install.ElasticsearchServiceStartAction.ElasticsearchServiceStart
Executing start service
Trying to execute StartServiceTask seeing service: True
Elastic.InstallerHosts.Elasticsearch.Tasks.Install.StartServiceTask: Elasticsearch service
[0] waiting to start waiting to start
[0] waiting to start waiting to start
... (repeated) ....
[0] waiting to start waiting to start
[0] waiting to start waiting to start
System.ServiceProcess.TimeoutException: Timeout has expired and the operation has not been completed in 00:01:00.
   at Elastic.Installer.Domain.Configuration.Service.ServiceStateProvider.StartAndWaitForRunning(TimeSpan timeToWait, Int32 totalTicks)
   at Elastic.InstallerHosts.Elasticsearch.Tasks.Install.StartServiceTask.ExecuteTask()
   at Elastic.InstallerHosts.SessionExtensions.Handle(Session session, Func`1 action)
CustomAction ElasticsearchServiceStartAction returned actual error code 1603 (note this may not be 100% accurate if translation happened inside sandbox)
Action ended 19:22:17: InstallFinalize. Return value 3.
```

We have been unable to replicate this issue in integration tests however.

**Comment by russcam** [comment]
A change has been made in https://github.com/elastic/windows-installers/pull/141 to the way in which the Windows Service is installed and started. This change is scheduled to go into `6.1.1`

**Comment by russcam** [comment]
As a workaround, if you experience this issue, you can

1. Opt to install Elasticsearch as a service
2. Opt to not start as part of the installation.
3. Start the service from the Services control panel or sc.exe, after installation

If an issue is encountered and the Elasticsearch service does not remain started. check the elasticsearch.log file in the logs directory location.

**Comment by codebrain** [comment]
Closing as believe issue is addressed in #141
